wled00/wled.h

```diff
@@ -177,6 +177,7 @@ using PSRAMDynamicJsonDocument = BasicJsonDocument<PSRAM_Allocator>;
 #define PSRAMDynamicJsonDocument DynamicJsonDocument
 #endif
 
+#include "FastLED.h"
 #include "const.h"
```

Comment:
To avoid inconsistent configured fastled instances, it's better to use the same fastled flags as in FX.h

https://github.com/Aircoookie/WLED/blob/2b0fad87d21e3a1f5538b8e238e92594d303da9c/wled00/FX.h#L34-L36